metrics/prometheus-metrics/src/main/scala/sttp/tapir/server/metrics/prometheus/PrometheusMetrics.scala

```diff
@@ -82,57 +85,57 @@ object PrometheusMetrics {
       )
     )
 
-  def requestActive[F[_]](registry: CollectorRegistry, namespace: String, labels: MetricLabels): Metric[F, Gauge] =
+  def requestActive[F[_]](registry: PrometheusRegistry, namespace: String, labels: MetricLabels): Metric[F, Gauge] =
     Metric[F, Gauge](
       Gauge
-        .build()
-        .namespace(namespace)
-        .name("request_active")
+        .builder()
+        .name(metricNameWithNamespace(namespace, "request_active"))
         .help("Active HTTP requests")
         .labelNames(labels.namesForRequest: _*)
-        .create()
         .register(registry),
       onRequest = { (req, gauge, m) =>
         m.unit {
           EndpointMetric()
-            .onEndpointRequest { ep => m.eval(gauge.labels(labels.valuesForRequest(ep, req): _*).inc()) }
-            .onResponseBody { (ep, _) => m.eval(gauge.labels(labels.valuesForRequest(ep, req): _*).dec()) }
-            .onException { (ep, _) => m.eval(gauge.labels(labels.valuesForRequest(ep, req): _*).dec()) }
+            .onEndpointRequest { ep => m.eval(gauge.labelValues(labels.valuesForRequest(ep, req): _*).inc()) }
+            .onResponseBody { (ep, _) => m.eval(gauge.labelValues(labels.valuesForRequest(ep, req): _*).dec()) }
+            .onException { (ep, _) => m.eval(gauge.labelValues(labels.valuesForRequest(ep, req): _*).dec()) }
         }
       }
     )
 
-  def requestTotal[F[_]](registry: CollectorRegistry, namespace: String, labels: MetricLabels): Metric[F, Counter] =
+  def requestTotal[F[_]](registry: PrometheusRegistry, namespace: String, labels: MetricLabels): Metric[F, Counter] =
     Metric[F, Counter](
       Counter
-        .build()
-        .namespace(namespace)
-        .name("request_total")
+        .builder()
+//        .namespace(namespace)
```

Comment:
since the namespace is included in the name, these should be removed I suppose?

Reply:
oh. I forgot to remove that comment. Looks like the new client doesn't have a `namespace` method. That's why I created `metricNameWithNamespace`